### PR TITLE
OO_#25 adding the description of the abotyp

### PR DESCRIPTION
**Pull request by mcmpp**

### CODE REVIEW

File: app/scripts/abos/list/aboslist.html [hunk 1]
```diff
@@ -29,6 +29,14 @@ <h3 class="panel-title"><span class="navbar-brand"><span class="small">{{abo.id}
              {{abo.vertrieb.beschrieb| translate}}
           </div>
         </div>
+        <div class="row vertrieb" ng-if="abo.vertrieb.beschrieb">
```
Comment (AlwinEgger) [inline app/scripts/abos/list/aboslist.html]: can you replace the css class `vertrieb` by `abotyp`? Attributing different classes to each sections makes it possible to hide sections using custom CSS easily.

File: app/scripts/abos/list/aboslist.html [hunk 2]
```diff
@@ -29,6 +29,14 @@ <h3 class="panel-title"><span class="navbar-brand"><span class="small">{{abo.id}
...
+          <div class="col-md-4 col-sm-5 col-xs-5" >
+            <span translate>Beschreibung:</span>
```
Comment (AlwinEgger) [inline app/scripts/abos/list/aboslist.html]: `Beschreibung:` is too generic in my opinion. What do we describe? So maybe `Abo-Beschrieb:` (to be almost as short)

**Review by AlwinEgger — DISMISSED** [review]

File: app/scripts/abos/list/aboslist.html (hunk 1 above)
Comment (mcmpp) [reply app/scripts/abos/list/aboslist.html]: done

File: app/scripts/abos/list/aboslist.html (hunk 2 above)
Comment (AlwinEgger) [reply app/scripts/abos/list/aboslist.html]: Hmm... You marked this as resolved but the changed tackles the other remark I made...

Comment (mcmpp) [reply app/scripts/abos/list/aboslist.html]: ups, I got confuse with the 2 requests. I reopen and apply the change

Comment (mcmpp) [reply app/scripts/abos/list/aboslist.html]: done

**Review by rawyler — APPROVED** [review]

**Pull request merged**